extra/resources/HealthCPUIoWait

```diff
@@ -0,0 +1,177 @@
+#!/bin/sh
+#
+# CPUIOWait RA
+# Measures CPU iowait % via top and writes #health-cpuiowait status into the CIB
+#
+# Originally based on ocf:pacemaker:HealthCPU RA
+# Copyright 2004-2009 SUSE LINUX AG, Lars Marowsky-Brée. All Rights Reserved.
+#
+# Later changes copyright 2009-2018 Michael Schwartzkopff
+# in collaboration with the Bull company. Merci! All Rights Reserved.
+#
+# Later changes copyright 2018 Maciej Sobkowiak <asm@egnyte.com>
+#
+# This source code is licensed under the GNU General Public License version 2
+# (GPLv2) WITHOUT ANY WARRANTY.
+#
+#######################################################################
+# Initialization:
+
+: ${OCF_FUNCTIONS=${OCF_ROOT}/resource.d/heartbeat/.ocf-shellfuncs}
+. ${OCF_FUNCTIONS}
+: ${__OCF_ACTION=$1}
+
+#######################################################################
+
+meta_data() {
+        cat <<END
+<?xml version="1.0"?>
+<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
+<resource-agent name="CPUIoWait" version="1.0">
+<version>1.0</version>
+
+<longdesc lang="en">
+Systhem health agent that measures the CPU iowait via top and updates the #health-cpuiowait attribute.
+</longdesc>
+<shortdesc lang="en">System health CPU iowait measurement</shortdesc>
+
+<parameters>
+<parameter name="state" unique="1">
+<longdesc lang="en">
+Location to store the resource state in.
+</longdesc>
+<shortdesc lang="en">State file</shortdesc>
+<content type="string" default="${HA_VARRUN%%/}/health-cpuiowait-${OCF_RESOURCE_INSTANCE}.state" />
+</parameter>
+
+<parameter name="yellow_limit" unique="1">
```

Comment:
The unique attribute has more than one use, but since health agents don't need a reload action, I think it makes sense to use its original purpose of indicating whether there can be multiple resource instances with the same value. In other words, it makes sense that "state" is unique (each resource instance must have its own state file), but "yellow_limit" and "red_limit" do not need to be unique.

Reply:
Fixd.